src/main/java/ru/practicum/shareit/item/ItemServiceImpl.java

```diff
@@ -0,0 +1,83 @@
+package ru.practicum.shareit.item;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ru.practicum.shareit.exceptions.EntityNotFoundException;
+import ru.practicum.shareit.exceptions.InternalServerException;
+import ru.practicum.shareit.item.dto.ItemResponseDto;
+import ru.practicum.shareit.item.dto.ItemRequestDto;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.user.UserService;
+
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class ItemServiceImpl implements ItemService {
+    private final ItemRepository itemRepository;
+    private final UserService userService;
+    private long id = 1L;
+
+    @Override
+    public ItemResponseDto postItem(ItemRequestDto itemRequestDto) {
+        log.debug("postItem. Добавление вещи ");
+        Item item = ItemMapper.mapToItem(itemRequestDto);
+        item.setId(getItemId());
+        userService.getUser(item.getOwner());
+        item = itemRepository.postItem(item).orElseThrow(() -> {
+            throw new InternalServerException("Ошибка записи");
```

Comment:
В методе `.orElseThrow()` уже заложена логика выброса исключения, поэтому явно определять `throw` не нужно

```suggestion
            new InternalServerException("Ошибка записи"));
```